### ISSUE TITLE
[Mobile] Show Veterans Pre-Need Burial Claims Status

### ISSUE THREAD

**Issue by jennb33** [department-of-veterans-affairs/va-mobile-feature-support#74]
#### Problem to be solved
Currently eligible Veterans engaging with the VA form-40-10007 (Application for Pre-need Determination of Eligibility for Burial in a VA National Cemetery) do not have a way to find, track and get statuses in the VA Flagship App and VA.gov web. This means the Veterans can only go through Va.gov to access this information.

In June 2017, VA launched VA form-40-10007 (Application for Pre-need Determination of Eligibility for Burial in a VA National Cemetery) on Vets.gov. Vets.gov later transitioned to Va.Gov which currently enables Veterans to discover, apply for, track and manage this claim.

As of today, current focus has only been on maintaining this feature. No planned work is currently scheduled. However all Burial products are expected to be shifted to a new team that is being stood up which will likely change current focus and future work.

Pre-need decision letter is not needed for a service member that is on active duty.

Pre-Need Burial Claims are not currently in the claims status tool, only burial claims are in the claims too.

The Flagship Mobile team did a review and analysis of this work [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/features/PreNeed%20Burial%20Claim/PreNeedBurialClaim.md), which should be used to plan this work. 

Specifically, we need to understand if the work outlined by Flagship Mobile back in 2023 is still viable and the most effective and efficient way to proceed. The strategy that was outlined is as follows: 

Product features for form-40-10007 (Application for Pre-need Determination of Eligibility for Burial in a VA National Cemetery) may include: 
- Discovery - ability to find their Pre-need claim in the flagship app
- Get Claim Updates - ability to tap their pre-need claim in the mobile app to get an update on their claim status
- View and download letter - ability to save a digital copy for storage and share it with necessary parties like family member, pre-planning funeral home director
- [Provide next steps after applying for eligibility](https://www.va.gov/burials-memorials/pre-need-eligibility/after-you-apply/)y - guide and educate Veteran on what they can do next
- Provide next steps if their pre-needs burial was denied

## Definition of done
That veterans can see the status of their Pre-Need Burial Claim (VA form-40-10007) using the mobile app and/or VA.gov as appropriate.

[Product Outline](https://docs.google.com/document/d/1WE9s6Mzy0sgtb1qAKAkTDLFHRVWG_FLesqfpL4PDZA0/edit?tab=t.0#heading=h.b2350v21tsmd)

## High Level User Story/ies
As the MFS team, 
We need to review and begin work on the Pre-Need Burial Claim, 
So that this can be added to the Mobile App interface for the benefit of Veterans.

MFS Product Brief is [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/pre-need/mobile/product/product-brief.md).
 
#### Monthly updates:
| Date | Responsible team | Task | Status | Notes |
| --- | --- | --- | --- | --- |
| December 2024 | MFS | December 2024 work includes: Reviewing research completed to date, confirming scope of MVP work, coming up with a Research Plan for any future work | Complete | | 
| | MFS | Work completed in December 2024 included: SPIKE tickets for UX and Eng to familiarize themselves with research and work done by Core Mobile Team  | Complete | | 
| January 2025 | MFS | January 2025 goals include completion of the research plan, any design needs, and to update the user flow for new elements | Complete | | 
| January 2025 | MFS | January 2025 work included circulating the Research plan internally and confirming API work in anticipation of development | Complete | | 
| February 2025 | MFS | February 2025 work included determining recruitment & screeners, completed prototype, and conversation guide for user interviews | Complete | | 
| March 2025 | MFS | March 2025 work included conducting the pre-need burial user interviews | Complete | | 
| April 2025 | MFS | April 2025 work included synthesizing research, developing research artifacts, and beginning to present research findings | In Progress | | 

| VA Stakeholders | Title | Role | Email | 
| --- | --- | --- | --- | 
| Jeff Barnes | Digital Experience Portfolio | Product Owner | [jeffrey.barnes4@va.gov](mailto:jeffrey.barnes4@va.gov) |
| Ryan Thurlwell | Mobile | Design Lead | [ryan.thurlwell@va.gov](mailto:ryan.thurlwell@va.gov) |
| Don McCaughey | Mobile | Tech Lead | [Donald.McCaughey@va.gov](mailto:Donald.McCaughey@va.gov) |

**OCTO lead** 
Jeff Barnes, Digital Experience Portfolio, Product Owner

**Is this initiative tracked to an upcoming event or congressional deadline?** - 
No

## Details

**Which OCTO team should do this? When might they have time to do this?** 
This work is on the MFS team board to be completed. Work is currently in flight, with an anticipated completion date of June 1, 2025 _(date is to be confirmed once the entire scope of work is understood)_

**Which OCTO or portfolio priority does this work support?** 
## OKR
Objective 2: Our platforms are the best way to delivery products at the VA. 
OKR 2: Our platforms measure and improve on the satisfaction of the user. 

**What is the user impact and scale of impact?** 
Once the Pre-Needs Burial Claim VA form-40-10007 has been implemented as a Mobile feature, Veterans can apply and see the status of their claim on the current Mobile platform before the Burial is needed.  

**What's the long-term plan for managing this effort?** 
Standard operations 

**What funding/contract support would this need?** 
Unknown at this time.  

**What else do we need to know in order to decide if/when we can do this work?**
Work is already in flight, as it had been prioritized by OCTO prior to this team's existence. 

## Reference links:
MFS Product Brief is [here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/burials-memorials/pre-need/mobile/product/product-brief.md).
MFS Research Plan is [here](https://docs.google.com/document/d/1UJHkDRLuAz-wbTyel18ii9oflTONdwWB22OEl4U7bHA/edit?tab=t.0)

## Hypothesis or Bet
If Pre-Need Burial Claims is completed, then more Veterans can find out about their eligibility and view the results of their Pre-Needs Burial Claim VA form-40-10007 (Application for Pre-need Determination of Eligibility for Burial in a VA National Cemetery) on the Mobile App and/or VA.gov as appropriate.

## Desired Outcomes
Enable Veterans to see the status of their Pre-Needs Burial Claim VA form-40-10007 (Application for Pre-need Determination of Eligibility for Burial in a VA National Cemetery) in the VA Mobile App and/or VA.gov as appropriate.

*Take into consideration Accessibility/QA needs as well as Product, Technical, and Design requirements.*

**Comment by mmiddaugh** [comment]
Currently, the only feasible mobile app functionality is the submission of pre-need eligibility forms. No development is planned for the service that provides pre-need burial data, as that system (EOAS) is being phased out. EOAS is expected to be replaced later this year by CaMEO, a Salesforce-based service. Once CaMEO is in place, we can reassess which APIs will be available for further development.
